crates/bevy_math/src/rotation2d.rs

```diff
@@ -331,10 +331,20 @@ impl Rot2 {
 
     /// Returns the angle in radians needed to make `self` and `other` coincide.
     #[inline]
+    #[deprecated(
+        since = "0.15.0",
+        note = "Use angle_to() instead, the semantics of angle_between will change in the future."
```

Comment:
We generally use backticks to distinguish variables and methods and things in comments (and presumably warnings), as per the [style_guide](https://bevyengine.org/learn/contribute/helping-out/opening-pull-requests/#style-guide)
```suggestion
        note = "Use `angle_to` instead, the semantics of `angle_between` will change in the future."
```